lexicons/com/atproto/admin/searchAccounts.json

```diff
@@ -0,0 +1,45 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.admin.searchAccounts",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get list of accounts that matches your search query.",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "email": { "type": "string" },
+          "limit": { "type": "integer" },
+          "cursor": { "type": "string" }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["accounts"],
+          "properties": {
+            "cursor": { "type": "string" },
+            "accounts": {
+              "type": "array",
+              "items": {
+                "type": "ref",
+                "ref": "#accountSearchResult"
+              }
+            }
+          }
+        }
+      }
+    },
+    "accountSearchResult": {
+      "type": "object",
+      "required": ["did"],
+      "properties": {
+        "did": { "type": "string" },
+        "email": { "type": "string" },
+        "normalizedEmail": { "type": "string" },
```

Comment:
I think of the `normalizedEmail` as an implementation detail of how our email search functionality works.  If we don't have a strong need for it in the client, or if the client can normalize the email itself for the user, I wonder if that may be preferable.